laravel/app/Providers/AppServiceProvider.php

```diff
@@ -14,7 +17,8 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+        $this->app->bind(RequestLoggerInterface::class, LocalRequestLogger::class); 
```

Comment:
Please use condition binding providing callback instead of className for `concrete`